ariadne_codegen/settings.py

```diff
@@ -74,9 +92,10 @@ class ClientSettings(BaseSettings):
     scalars: Dict[str, ScalarData] = field(default_factory=dict)
 
     def __post_init__(self):
+        super().__post_init__()
```

Comment:
This change is breaking the tests. Please review and fix the issue to ensure the tests pass.